src/schema/search.ts

```diff
@@ -331,6 +331,7 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         .where('post.id IN (:...ids)', {
           ids: hits.length ? hits.map((x) => x.post_id) : ['nosuchid'],
         })
+        .andWhere('post.title IS NOT NULL')
```

Comment:
Wondering if this will slow down the query since we don't have the index on `title`, but maybe we can achive the same with just `filter` on JS side.

Reply:
That's also fine with me, we can do it in the map below if you prefer?

Reply:
I check with `explain analyze`, it should be good since we filter by id either way which is PK.